entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+python3 -m http.server 8080 --directory /iglu-schemas &
+
+/opt/docker/bin/snowplow-micro $@
```

Comment:
Most entrypoints I see end with `exec /some/binary $@` and there are some comments [in this stack overflow post](https://stackoverflow.com/a/48096779/11306550) explaining why.

Reply:
Great, thanks for the tip!